vendor/assets/stylesheets/dvl/core/grid.scss

```diff
@@ -11,6 +11,9 @@
   padding: 0;
   list-style: none; /* 2 */
   font-size: 0; /* 3 */
+  @media screen and (min-width: $lapWidth) {
```

Comment:
Are we no longer adjusting our base font size based on viewport? Because if so, then that's a whole other issue...